### ISSUE TITLE
Can't install Package Control on Sublime Text 3

### ISSUE THREAD

**Issue by Revod** [wbond/package_control#993]
I Installed a fresh version of Elementary OS (based on ubuntu 14.04) i have a old pc with 2gb of ram but when i pu the code in the  console i get error, attach spect  and error above:

Elementary OS Freya (Based on Ubuntu 14.04) 32 bit
2gb ram
Sublime Text 3 Build 3083

I Put the code https://packagecontrol.io/installation#st3 in the console and get:

```
>>> import urllib.request,os,hashlib; h = '462d6673ebd5b9ca297474c166b2e10d' + 'fd945a23b1a0642535aa906951315f52'; pf = 'Package Control.sublime-package'; ipp = sublime.installed_packages_path(); urllib.request.install_opener( urllib.request.build_opener( urllib.request.ProxyHandler()) ); by = urllib.request.urlopen( 'http://packagecontrol.io/' + pf.replace(' ', '%20')).read(); dh = hashlib.sha256(by).hexdigest(); print('Error validating download (got %s instead of %s), please try manual install' % (dh, h)) if dh != h else open(os.path.join( ipp, pf), 'wb' ).write(by)
186933
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
reloading Packages/Package Control/Package Control.sublime-settings
Package Control: Downloading bootstrapped dependency ssl-linux
reloading plugin 0_package_control_loader.00-package_control
ignored packages updated to: ["Package Control", "Vintage"]
unloading plugin Package Control.1_reloader
unloading plugin Package Control.2_bootstrap
unloading plugin Package Control.Package Control
reloading Packages/Package Control/Package Control.sublime-settings
reloading Packages/User/Preferences.sublime-settings
Package Control: Settings change detected, clearing cache
Package Control: Installing 2 missing dependencies
Exception in thread Thread-3:
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "package_control.automatic_upgrader in /home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package", line 113, in run
  File "package_control.automatic_upgrader in /home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package", line 145, in install_missing
  File "package_control.package_manager in /home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package", line 864, in install_package
  File "package_control.package_manager in /home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package", line 453, in list_available_packages
  File "package_control.package_manager in /home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package", line 347, in list_repositories
TypeError: 'NoneType' object is not subscriptable

Package Control: Removed orphaned package Package Control
ignored packages updated to: ["Vintage"]
error loading /home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package: [Errno 2] No such file or directory: '/home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package'
reloading Packages/User/Preferences.sublime-settings
Package Control: Successfully downloaded bootstraped dependency ssl-linux
Exception in thread Thread-1:
Traceback (most recent call last):
  File "./threading.py", line 901, in _bootstrap_inner
  File "./threading.py", line 858, in run
  File "package_control.bootstrap in /home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package", line 181, in bootstrap_dependency
  File "package_control.loader in /home/revod/.config/sublime-text-3/Installed Packages/Package Control.sublime-package", line 207, in add
  File "/home/revod/.config/sublime-text-3/Installed Packages/0_package_control_loader.sublime-package/01-ssl-linux.py", line 6, in <module>
ImportError: cannot import name sys_path
```

**Comment by wbond** [comment]
This should be solved by version 3.1.2, which can now be installed by following the directions at https://packagecontrol.io/installation.

**Comment by Revod** [comment]
Thanks for the fix
